metadata_service/proxy/atlas_proxy.py

```diff
@@ -762,14 +759,33 @@ def _get_resources_owned_by_user(self, user_id: str, resource_type: str) \
             LOGGER.exception(f'User ({user_id}) not found in Atlas')
             raise NotFoundException(f'User {user_id} not found.')
 
-        resource_guids = list()
+        resource_guids = set()
         for item in user_entity[self.REL_ATTRS_KEY].get('ownerOf') or list():
             if (item['entityStatus'] == Status.ACTIVE and
                     item['relationshipStatus'] == Status.ACTIVE and
                     item['typeName'] == resource_type):
-                resource_guids.append(item[self.GUID_KEY])
+                resource_guids.add(item[self.GUID_KEY])
+
+        params = {
+            'typeName': self.TABLE_ENTITY,
+            'excludeDeletedEntities': True,
+            'entityFilters': {
+                'condition': 'AND',
+                'criterion': [
+                    {
+                        'attributeName': 'owner',
+                        'operator': 'contains',
```

Comment:
What if company has IDs in form of integers ? Wouldn't it cause some overlaps (for example table with owner=1 could clash with table with owner=10)